ui/apps/platform/src/Containers/SystemHealth/Components/IntegrationHealthWidgetVisual.tsx

```diff
@@ -1,40 +1,75 @@
+    Flex,
+    FlexItem,
+} from '@patternfly/react-core';
+import { CheckCircleIcon, ExclamationCircleIcon } from '@patternfly/react-icons';
 
-import { integrationsPath } from 'routePaths';
-import ViewAllButton from 'Components/ViewAllButton';
-import Widget from 'Components/Widget';
+import pluralize from 'pluralize';
+import { Message } from '@stackrox/ui-components';
 import IntegrationsHealth from './IntegrationsHealth';
 import { IntegrationMergedItem } from '../utils/integrations';
+import IconText from '../../../Components/PatternFly/IconText/IconText';
 
 type IntegrationHealthWidgetProps = {
-    id: string;
     integrationText: string;
     integrationsMerged: IntegrationMergedItem[];
     requestHasError: boolean;
 };
 
 const IntegrationHealthWidget = ({
-    id,
     integrationText,
     integrationsMerged,
     requestHasError,
 }: IntegrationHealthWidgetProps): ReactElement => {
+    const integrations = integrationsMerged.filter((integrationMergedItem) => {
+        return integrationMergedItem.status === 'UNHEALTHY';
+    });
     return (
-        <Widget
-            header={integrationText}
-            headerComponents={<ViewAllButton url={`${integrationsPath}#${id}`} />}
-            className="h-48 text-lg"
-            id={id}
-        >
-            {requestHasError ? (
-                <div className="p-2 w-full">
+        <Card isFullHeight isCompact>
+            <CardHeader>
+                <CardHeaderMain>
+                    <Flex alignItems={{ default: 'alignItemsCenter' }}>
+                        <FlexItem>
+                            <CardTitle component="h2">{integrationText}</CardTitle>
+                        </FlexItem>
+                        {!requestHasError && (
+                            <FlexItem>
+                                <IconText
+                                    icon={
+                                        integrations.length === 0 ? (
+                                            <CheckCircleIcon color="var(--pf-global--success-color--100)" />
+                                        ) : (
+                                            <ExclamationCircleIcon color="var(--pf-global--danger-color--100)" />
+                                        )
+                                    }
+                                    text={
+                                        integrations.length === 0
+                                            ? 'no errors'
+                                            : `${integrations.length} ${pluralize(
+                                                  'error',
+                                                  integrations.length
+                                              )}`
+                                    }
+                                />
+                            </FlexItem>
+                        )}
+                    </Flex>
+                </CardHeaderMain>
+            </CardHeader>
+            <CardBody>
```

Comment:
Not a blocker to merge, `return <></>` statement in the following file reminded me to show slightly different condition to omit `CardBody` element instead of render it with empty content:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/SystemHealth/DeclarativeConfigurationHealth/DeclarativeConfigurationHealthCard.tsx#L85

Although the difference is subtle, you can see the effect of empty `CardBody` as slightly more empty space under title of 3 integrations cards compared to no `CardBody` in **Vulnerability definitions** and **Declarative configuration** cards above and below in the second picture of healthy cards.